loader/windows/icd_windows_apppackage.cpp

```diff
@@ -129,7 +129,7 @@ extern "C" bool khrIcdOsVendorsEnumerateAppPackage()
         }
 
         HString path;
-        if (FAILED(item->get_Path(path.ReleaseAndGetAddressOf())))
+        if (FAILED(item->get_Path(path.GetAddressOf())))
```

Comment:
I assume this should ultimately be squashed with the first commit as well?

Reply:
Yes, this is a fixup to support older SDKs. I had no idea that function wasn't there in older SDKs.